web-app/js/HighDimensionalData.js

```diff
@@ -350,10 +350,12 @@ HighDimensionalData.prototype.get_inputs = function (divId) {
     ]
 }
 
-HighDimensionalData.prototype.gather_high_dimensional_data = function (divId, hideAggregration) {
+HighDimensionalData.prototype.gather_high_dimensional_data = function (divId, hideAggregration, doValidatePlatforms) {
```

Comment:
I do not see any call of this method with `doValidatePlatforms = true`. Are you sure you didn't disable platform validation for all analyses?